docs/docs/data-storage-redux.md

```diff
@@ -1,9 +1,51 @@
+- **[Static Query Components](/docs/static-vs-normal-queries/#keeping-track-of-site-queries-during-build-in-redux-stores)** - A `Map` of any components detected with a static query.
+- **Jobs** - Long-running processes, generally started as a side-effect to a GraphQL query. Gatsby doesn’t finish its process until all jobs are ended.
+- **Webpack** - Config for the [Webpack](https://webpack.js.org/) bundler which handles code optimization and splitting of delivered javascript bundles.
+
+The Gatsby [/redux index file](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby/src/redux/index.ts) has two key exports, `store` and `emitter`. Throughout the bootstrap and build phases, `store` is used to get the current state and dispatch actions, while `emitter` is used to register listeners for particular actions. The store is also made available to Gatsby users through the [Node APIs](/docs/node-apis/).
+
+## Actions
+
+Actions dispatched in the store cause state changes through the reducers and also trigger listeners registered for that action on a [mitt](https://www.npmjs.com/package/mitt) `emitter`. While the `subscribe` Redux store method is typically used to connect a web framework like React, Gatsby only uses the `subscribe` method to connect the `emitter`.
+
+The [Gatsby actions](/docs/actions/) are all either internal, public or restricted. The public actions, and a context relevant subset of the restricted actions, are available to users through the [Node APIs](/docs/node-apis/).
+
+### Action Journey
+
+Gatsby actions have a similar journey through defining, exposing and dispatching. Let's follow the [createRedirect](/docs/actions/#createRedirect) public action:
+
+- **Reducer case** - The redirects reducer will catch actions with a type `CREATE_REDIRECT` and make the necessary state manipulation.
+
+- **Side effect** - An `emitter` listener is registered for the `CREATE_REDIRECT` action type.
+
+- **Action creator** - An action creator, `createRedirect`, is defined in the public actions file. The action has a payload, the information needed to complete the action, and a type, the string that identifies this particular action.
+
+- **Expose bound action creator** - `createRedirect` is one of the public actions made available to all of the [Node APIs](/docs/node-apis/). A collection of public actions and the restricted actions available to the called API are bound to the Redux store dispatch. The bound action collection is then passed when calling the user's API function.
+
+- **Dispatch** - Here is an example of the `createRedirect` call that a Gatsby user could make with the [createPages](/docs/node-apis/#createPages) API in their project's [gatsby-node.js](/docs/api-files-gatsby-node/) file:
+
+```javascript:title=gatsby-node.js
+module.exports = {
+  createPages: ({ actions }) => {
+    const { createRedirect } = actions
+    createRedirect({
+      fromPath: "/legacy-path",
+      toPath: "/current-path",
+    })
+  },
+}
+```
```

Comment:
````suggestion
```

By walking through an action scenario in detail, you can hopefully understand more about Gatsby's internals using Redux.

## Additional resources

- [Behind the Scenes: What Makes Gatsby Great](/blog/2019-04-02-behind-the-scenes-what-makes-gatsby-great/)
- [Gatsby Magic](/docs/gatsby-magic/)
````

Upon reading the doc, it seemed like it needed a conclusion and some further reading. Let me know what you think!

Reply:
Agree on the conclusion sentence 👍

Hesitated on the additional resources though, reckon the best additional resource is the wider internals section that this doc is part of. They likely aready know that though, and these 2 pages are great and provide alternate angles on the internals.